mdtraj/geometry/order.py

```diff
@@ -189,6 +189,30 @@ def compute_inertia_tensor(traj):
     return A * eyes - B
 
 
+def compute_gyration_tensor(traj):
+    """Compute the gyration tensor of a trajectory.
+
+    For each frame,
+        S_{nm} = sum_{i_atoms} r^{i}_m r^{i}_n
+
+    Parameters
+    ----------
+    traj : Trajectory
+        Trajectory to compute gyration tensor of.
+
+    Returns
+    -------
+    S_xy:  np.ndarray, shape=(traj.n_frames, 3, 3), dtype=float64
+        Gyration tensors for each frame.
+
+    """
+    xyz = traj.xyz
+    center_of_geom = np.expand_dims(compute_center_of_geometry(traj), axis=1)
+    xyz -= center_of_geom
```

Comment:
i think this may change the coordinates in the trajectory object itself, which might be unexpected for users who wouldn't expect `compute_gyration_tensor` to modify the trajectory in place.

Reply:
Yup, I think this would would perhaps unwantingly modify the coordinates. Changed to more closely reflect how `compute_inertial_tensor` is currently written.